src/test/java/com/wikia/webdriver/common/dataprovider/ads/AdsDataProvider.java

```diff
@@ -379,7 +379,7 @@ private AdsDataProvider() {
             "adtest",
             "SyntheticTests/Async/Hop/ExtraMarker?log_level=9&log_group=Wikia.Tracker",
             "TOP_RIGHT_BOXAD",
-            "adType=async;method=hop;pos=TOP_RIGHT_BOXAD;test-marker=42"
+            "adType=async;method=hop;pos=TOP_RIGHT_BOXAD;sp=no;test-marker=42"
```

Comment:
Will it pass when event gets too late and there will be no `sp=no;`? How does the test look like? Let me take a look. Because instead of the string I'd go with regexp or... check if the output contains `adType=async;method=hop;pos=TOP_RIGHT_BOXAD;` and `test-marker=42`.